core/src/main/java/org/infinispan/manager/EmbeddedCacheManager.java

```diff
@@ -29,7 +29,7 @@
  * Lifecycle - <tt>EmbeddedCacheManager</tt>s have a lifecycle (it implements {@link Lifecycle}) and
  * the default constructors also call {@link #start()}.  Overloaded versions of the constructors are available, that do
  * not start the <tt>CacheManager</tt>, although it must be kept in mind that <tt>CacheManager</tt>s need to be started
- * before they can be used to create <tt>Cache</tt> instances.
+ * before they can be used to readWriteMap <tt>Cache</tt> instances.
```

Comment:
Looks like an inadvertent change?

Reply:
yes, a bad refactoring I did and I thought everything was "restored" but missed that